### PR TITLE
Fix  some mistake usage of LinearFunction.backward(...) and a argument of gradcheck(...) in the documents 

### PR DESCRIPTION
**Pull request by sccbhxc**

### CODE REVIEW

File: docs/source/notes/extending.rst [hunk 1]
```diff
@@ -56,11 +56,11 @@ additional comments::
         @staticmethod
         def backward(ctx, grad_output):
             # This is a pattern that is very convenient - at the top of backward
-            # unpack saved_tensors and initialize all gradients w.r.t. inputs to
```
Comment (zou3519) [inline docs/source/notes/extending.rst]: We renamed saved_variables as saved_tensors, so this change is not necessary

File: docs/source/notes/extending.rst [hunk 2]
```diff
@@ -56,11 +56,11 @@ additional comments::
...
+            # unpack saved_variables and initialize all gradients w.r.t. inputs to
             # None. Thanks to the fact that additional trailing Nones are
             # ignored, the return statement is simple even when the function has
             # optional inputs.
-            input, weight, bias = ctx.saved_tensors
```
Comment (zou3519) [inline docs/source/notes/extending.rst]: See above

**Review by zou3519 — CHANGES_REQUESTED** [review]
Thanks for the catch! It is intended that we're using `ctx.saved_tensors` instead of `ctx.saved_variables.`

**Comment by SsnL** [comment]
@sccbhxc Thanks for your PR. This is good to merge once the minor nit is fixed.

**Comment by zou3519** [comment]
https://github.com/pytorch/pytorch/pull/7725 fixes this, so I'm closing this PR.

**Pull request closed without merging**